pallets/token-manager/src/lib.rs

```diff
@@ -339,6 +354,27 @@ pub mod pallet {
     #[pallet::getter(fn lowers_disabled)]
     pub type LowersDisabled<T: Config> = StorageValue<_, bool, ValueQuery>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn next_burn_at)]
+    pub type NextBurnAt<T: Config> = StorageValue<_, BlockNumberFor<T>, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn burn_submission)]
+    pub type PendingBurnSubmission<T: Config> =
+        StorageMap<_, Blake2_128Concat, u32, U256, OptionQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn burn_refresh_range)]
+    pub type BurnRefreshRange<T> = StorageValue<_, u32, ValueQuery, DefaultBurnRefreshRange<T>>;
```

Comment:
Does it make sense to rename this to `BurnPeriod` or something similar? It will also match the extrinsic